site/en/docs/webstore/cws-dashboard-privacy/index.md

```diff
@@ -83,6 +83,12 @@ group of checkboxes to certify that you comply with each of the disclosure state
 {% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/PwmNuP356TsVi80bxWAj.png",
   alt="Screenshot of data use certification fields", width="800", height="495" %}
 
+## Set a privacy policy {: #set-privacy-policy }
+
+Add a link to the privacy policy for all your extensions. The policy should include how data is collected, used, and disclosed. See the [User Data FAQ][user-data] for more details.
```

Comment:
This wording sounds a bit strange, since you are editing the information for one item on the privacy tab.

I would say something like: "Add a link to the privacy policy of your extension"